pkg/model/components/etcdmanager/model.go

```diff
@@ -258,7 +258,7 @@ func (b *EtcdManagerBuilder) buildPod(etcdCluster kops.EtcdClusterSpec) (*v1.Pod
 
 	etcdInsecure := !b.UseEtcdTLS()
 
-	clientHost := "__name__"
+	clientHost := etcdCluster.Name + ".etcd." + b.ClusterName()
```

Comment:
same as above, consider using MasterPublicName or MasterInternalName API field here?

Reply:
We didn't use `__name__` for the cilium etcd cluster. So this sort of just followed that logic.